src/TextRight.Editor.Wpf/View/DocumentCursorView.cs

```diff
@@ -214,7 +225,7 @@ private bool MeasureCursor()
     private IndexLayout GetCurrentLayoutIndex()
     {
       // TODO how can we do this not having so many 'as' casts?
-      var associatedView = (_cursor.Start.Block as IDocumentItem)?.DocumentItemView as ILayoutable;
+      var associatedView = (_cursor.Start.Block as IDocumentItem)?.Tag as ILayoutable;

```

Comment:
I think `GetView` would be better + would get rid of the `as` casts comment above